.npmignore

```diff
@@ -0,0 +1,15 @@
+.github/
+.husky/
+test/
+.commitlintrc.yml
+.eslintignore
+.eslintrc.json
+.gitignore
+.lintstagedrc.yml
+.prettierrc.yml
+.releaserc.yml
+jest.config.ts
+Dockerfile
+tsconfig.build.json
+tsconfig.json
```

Comment:
If we don't have tests for this package, then the whole tsconfig.build file is not necessary. You can rename it, remove the other file and remothe the paths and baseUrl options (or update them if you want to use base path settings)